cernopendata/templates/cernopendata_theme/header.html

```diff
@@ -9,25 +9,14 @@
   </button>
 
   <div class="collapse navbar-collapse" id="navbarSupportedContent">
-
-    <form {% if request.endpoint=='cernopendata_pages.index' %} id="main-search" {% endif %}
-    class="form-inline my-2 my-lg-0" action="{{config.THEME_SEARCH_ENDPOINT}}">
-    <div class="input-group w-100">
-      <input class="form-control" type="text" placeholder="Search" name="q" aria-label="Search"
-             value="{{ request.args.get('q','')}}">
-      <span class="input-group-btn">
-        <button class="btn btn-info" type="submit"><span class="oi" data-glyph="magnifying-glass"></span>
-        </button>
-        </span>
-    </div>
-    </form>
     <img id="demo-logo" src="{{url_for('static', filename='img/demo.png')}}" alt=""
```

Comment:
Nice home page! 👍 

One substantial comment: while it is good to remove the search input box in the page header, as you did here,  please add some rather *prominent* search box on the `/search` pages, otherwise people won't see what they typed... The search box can be positioned centered above both "Fiter by type" and "Sory by" for the time being...

![search](https://user-images.githubusercontent.com/517546/33531803-a366b672-d892-11e7-8281-37d94551e2de.png)